app/serializers/email_serializer.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class EmailSerializer < ActiveModel::Serializer
+  attribute :email
+  attribute :effective_at
+
+  def id
+    nil
+  end
+
+  # Returns the email address nested in the given object.
+  #
+  # @return [String] Email address.  Sample `object.email_address`:
+  #   {
+  #     "effective_date" => "2018-02-27T14:41:32.283Z",
+  #     "value" => "test2@test1.net"
+  #   }
+  #
+  def email
+    object&.email_address&.dig 'value'
+  end
+
+  # Returns the email's effective datetime nested in the given object
+  #
+  # @return [String] Effective datetime in the yyyy-MM-dd'T'HH:mm:ss format.
+  #   Sample `object.email_address`:
+  #     {
+  #       "effective_date" => "2018-02-27T14:41:32.283Z",
+  #       "value" => "test2@test1.net"
+  #     }
+  #
+  def effective_at
+    object&.email_address&.dig 'effective_date'
+  end
```

Comment:
These kinds of mapping translations aren't ideal:
```
value --> email
effective_date ---> effective_at
```
If at all possible, reach out to EVSS and try to come to an agreement on attribute names.

Reply:
Good call.

@jkassemi - What would be the best way to get this underway?  Do we have a contact?

Also, perhaps we should get the ball rolling now, and then implement the changes in a follow up PR, so we don't delay this one's release?

Reply:
I'm happy with that, but only since we're not in production (otherwise the naming change on the EVSS side could cause downtime during migration). 

Can you reach out with this suggestion in the DSVA #evss slack channel? I'd like to make sure we get you introduced to the team. We can have a discussion there and then pass off to Matt Self to coordinate the change.

If we have trouble getting the name changed on the EVSS side, I think we should generally support the names that we believe are best for consumers of our API.